### ISSUE TITLE
[Wiki edit]  Oracle Cloud hosting

### ISSUE THREAD

**Issue by emhl** [jagrosh/MusicBot#754]
There are many great tips on where to host your own (not just music-) bot on your wiki.
Oracles cloud hosting service with their [Always free Program](https://www.oracle.com/cloud/free/) could be a great addition to the [hosting wiki page](https://github.com/jagrosh/MusicBot/wiki/24-7-Hosting) 

with this you get:
- 4 Arm-based Ampere A1 cores
- 24 GB of Ram

that can be split up however you want.
additionally you get  2 AMD based Compute VMs with 1/8 OCPU and 1 GB memory each.

everything completely free. the VM'shave well enough performance to host multiple instances of JMusicBot

the only small contra is, that you need to give them your credit card details.

**Comment by YJSoft** [comment]
Some things to consider:

1. Oracle's A1 Instance will be deleted after 30-day trial period, although it's available as "Always-Free". You should delete instance(but not deleting boot volume), and recreate using previous boot volume.
2. "Out of capacity" error might occur while try to create A1 instance. A1 instance server are limited, so when that error happens, you have to wait.(No ETA)

p.s.) It's not a good idea to use Micro E1 server to serve bots, as bandwith is limited to 50Mbps(A1 bandwith limits are high enough for bot)

**Comment by Sanduhr32** [comment]
50Mbps is more than enough for selfhosting this bot, because you ain't serving hundreds or thousands of music - streams in parallel.

**Comment by YJSoft** [comment]
I tried, and got terrible audio quality. Maybe slow cpu is main problem(since its limited to 1/8 OCPU)

**Comment by emhl** [comment]
@YJSoft audio quality is fine on my Micro instance and cpu usage peaked at 27% of the 1/8 Ocpu for a short time when i'm using the bot
is the datacenter you chose in the same Region as you and the discord server?

**Comment by YJSoft** [comment]
Yep. it was same region. Maybe my internet was bad, but I don't run JMusicBot anymore so...

Anyway, I still recommend A1 rather than Micro E1 since it's more powerful.

**Comment by krux3r** [comment]
I've been running the bot on Oracle for few days now, everything is working fine except bot goes offline during night, but the process stays running. Any thoughts about that?

**Comment by MichailiK** [comment]
That sounds like a re-connection issue. If you have the logs, you may open a new issue with the logs & output from the `debug` command

**Comment by MichailiK** [comment]
There actually seems to be an issue where the bot dies during prolonged inactivity. See #823

**Comment by MichailiK** [comment]
There are now reports of people getting suspended, https://github.com/jagrosh/MusicBot/pull/901#issuecomment-926343793 and https://github.com/jagrosh/MusicBot/pull/901#issuecomment-926356767. Because of these, this should probably be put on hold for now.

**Comment by emhl** [comment]
yeah, i've seen quite alot of reports like that over the last few weeks as well. I'll reopen the issue when no new reports of those suspensions get published anymore.

**Comment by emhl** [comment]
I haven't seen any reports of accounts getting terminated out of the blue on r/oraclecloud since the 17th of october, but it's probably better to wait a month longer to reopen this :D
